### PR TITLE
basic notebook tests

### PR DESCRIPTION
**Pull request by heejaechang**
added basic notebook sanity check test files

### CODE REVIEW

**Review by rchiodo — APPROVED** [review]

File: testing/single/src/showCallHierarchy.py [hunk 1]
```diff
@@ -48,8 +48,10 @@ def method(self):
 class BaseConsumer:
     def consumer(self, base: Base):
         base.method()
+        return base
```
Comment (StellaHuang95) [inline testing/single/src/showCallHierarchy.py:51]: Why is this added?

Comment (heejaechang) [reply testing/single/src/showCallHierarchy.py:51]: see `#test basic call hierarchy ...` cell.
